yml/3rd_party/avast/wsc.yml

```diff
@@ -5,6 +5,10 @@ Created: 2022-08-15
 Vendor: Avast
 ExpectedLocations:
   - '%PROGRAMFILES%\AVAST Software\Avast'
+ExpectedVersionInformation:
+  - OriginalName: wsc.dll
+  - InternalName: wsc
+  - FileDescription: Avast security center dll
```

Comment:
Is it worth trying to assert each entry under ExpectedVersionInformation MUST have all three `OriginalName`, `InternalName` and `FileDescription` specified, even when one or more are blank? This will prevent others making similar mistakes

Reply:
I think this is fair too